packages/code-du-travail-nlp/Dockerfile

```diff
@@ -5,11 +5,8 @@ ENV FLASK_APP ./api/app
 
 WORKDIR ${APP_DIR}
 
-COPY requirements.txt .
+COPY . .
```

Comment:
Better to be explicit about what you want to copy (for performance too)

Reply:
c'est ok si j'ai rajouté un `.dockerignore`   
et je 
```
COPY requirements.txt .
COPY api .
```